apps/tangle-dapp/src/context/bridge/useBridgeStore.ts

```diff
@@ -111,11 +111,12 @@ const useBridgeStore = create<BridgeStore>((set) => ({
         ].supportedTokens;
 
       return {
-        selectedSourceChain: chain,
-        destinationChains: availableDestinations,
-        selectedDestinationChain: availableDestinations[0],
-        tokens,
-        selectedToken: tokens[0],
+        ...state,
```

Comment:
We don't need the spread here; the setter returns an object that will be shallowly merged with the existing state in the store, read more [here](https://zustand.docs.pmnd.rs/guides/updating-state#flat-updates).